heron/config/src/yaml/conf/aurora/heron.aurora

```diff
@@ -20,7 +20,7 @@ fetch_user_package = Process(
   cmdline = 'curl %s -o %s && tar zxf %s' % (heron_topology_jar_uri, topology_package_file, topology_package_file)
 )
 
-command_to_start_executor = '{{EXECUTOR_BINARY}} {{mesos.instance}} {{TOPOLOGY_NAME}} {{TOPOLOGY_ID}} {{TOPOLOGY_DEFINITION_FILE}} {{STATEMGR_CONNECTION_STRING}} {{STATEMGR_ROOT_PATH}} {{TMASTER_BINARY}} {{STMGR_BINARY}} "{{METRICSMGR_CLASSPATH}}" {{INSTANCE_JVM_OPTS_IN_BASE64}} "{{TOPOLOGY_CLASSPATH}}" {{thermos.ports[port1]}} {{thermos.ports[port2]}} {{thermos.ports[port3]}} {{SYSTEM_YAML}} {{COMPONENT_RAMMAP}} {{COMPONENT_JVM_OPTS_IN_BASE64}} {{TOPOLOGY_PACKAGE_TYPE}} {{TOPOLOGY_BINARY_FILE}} {{JAVA_HOME}} {{thermos.ports[http]}} {{SHELL_BINARY}} {{thermos.ports[port4]}} {{CLUSTER}} {{ROLE}} {{ENVIRON}} "{{INSTANCE_CLASSPATH}}" {{METRICS_YAML}} "{{SCHEDULER_CLASSPATH}}" "{{thermos.ports[scheduler]}}" {{PYTHON_INSTANCE_BINARY}} {{METRICSCACHEMGR_CLASSPATH}} {{thermos.ports[metricscachemgr_masterport]}} {{thermos.ports[metricscachemgr_statsport]}} {{IS_STATEFUL_ENABLED}} "{{CKPTMGR_CLASSPATH}}" {{thermos.ports[ckptmgr_port]}} {{STATEFUL_CONFIG_YAML}}'
```

Comment:
Ok, please let me know once the private env is ready.

Also, I think it is time to make the `executor` smart. As per the current pattern, the scheduler loads the configs and passes to the `executor`. This could be avoided as the executor has access to the config files. Its a separate PR/discussion though. WDYT?